modules/apps/foundation/frontend-theme/responsive-boat-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,278 @@
+/* These inject tags are used for dynamically creating imports for themelet styles, you can place them where ever you like in this file. */
+
+/* inject:imports */
+/* endinject */
+
+/* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
+
+
+/********** VARIABLE SECTION **********/
+$expandTime: 0.4s; 				// time it takes for mobile navbar to expand or collapse
+$fadeTime: 0.5s; 				// time it takes for navbar links to change color when hovered over
+$navBtnPad: 20px; 				// padding above and below nav-button
+$navBtnHeight: 40px; 			// height of nav-button
+/********** END OF VARIABLE SECTION **********/
+
+
+
+/* All styling is nested within #wrapper selector to avoid accidentally styling the liferay menus */
+#wrapper {
+	padding: 0; 				// eliminate white spaces on edges of webpage
+	text-align: center; 		// default text-alignment to center all child texts and elements
+	background: {
+		color: darkgrey; 		// default background when image is not loaded
+		image: url("../images/background.jpg");
+		size: cover;
+		position-x: center;
+		attachment: fixed; 		// background stays fixed while scrolling
```

Comment:
We should spell out each CSS rule or short hand it instead of defining everything within this block.